docs/machine-learning/basic-concepts-model-training-in-mldotnet.MD

```diff
@@ -0,0 +1,169 @@
+---
+title: Basic concepts for model training in ML.NET
+description: This article explains the basic concepts covering *model training* scenarios in ML.NET. Not all concepts are relevant to the more simple scenario of *prediction with existing model*.
+ms.date: 11/06/2018
+ms.topic: overview
+ms.custom: mvc
+#Customer intent: As a developer, I want to learn the basic ML.NET concepts related to model training so that I can understand how to leverage machine learning in my applications.
+---
+# Basic concepts for model training in ML.NET
+
+This article explains the basic concepts covering *model training* scenarios in ML.NET. Not all concepts are relevant to the more simple scenario of *prediction with existing model*.
+
+## Model training concepts
+
+This document covers the following concepts related to model training in ML.NET:
+
+- [*Data*](#data), represented as an `IDataView` interface.
+  - In ML.NET, data is very similar to a SQL view: it's a lazily-evaluated, immutable, cursorable, heterogenous, schematized dataset. 
+  - An excellent document about the data interface is [IDataView Design Principles](IDataViewDesignPrinciples.md).
```

Comment:
This refers to the IDataViewDesignPrinciples doc in the repo: https://github.com/dotnet/machinelearning/blob/master/docs/code/IDataViewDesignPrinciples.md

Reply:
Is there an action you'd like here? I took this as is for now -- without edits.

Reply:
Nevermind. I see.